autoscheduler/frontend/src/utils/timeUtil.ts

```diff
@@ -9,9 +9,17 @@ export const LAST_HOUR = 22;
  * @param use24Hour if true, applies 24-hour format instead of the default 12-hour format
  */
 export function formatTime(h: number, m: number, use24Hour = false, padZeroes = false): string {
+  let amOrpm = '';
+  if (!use24Hour) {
+    amOrpm = 'AM';
+    if (h >= 12) {
+      amOrpm = 'PM';
+    }
```

Comment:
Might as well switch this to an if/else or ternary instead of always making it AM first